pkgs/by-name/re/resolute/package.nix

```diff
@@ -0,0 +1,88 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  rustPlatform,
+  buildNpmPackage,
+  npmHooks,
+  wrapGAppsHook4,
+  autoPatchelfHook,
+  nodejs,
+  rustc,
+  cargo,
+  cargo-tauri,
+  pkg-config,
+  openssl,
+  alsa-lib,
+  gdk-pixbuf,
+  atk,
+  cairo,
+  pango,
+  glib-networking,
+  webkitgtk_4_1,
+
+  xrSources,
+}:
+
+rustPlatform.buildRustPackage rec {
+  inherit (xrSources.resolute)
+    pname
+    version
+    src
+    date
+    ;
+  cargoLock = xrSources.resolute.cargoLock."Cargo.lock";
+
+  useFetchCargoVendor = true;
+
+  frontend = buildNpmPackage {
+    inherit version src;
+    pname = "Resolute-ui";
+
+    npmDepsHash = "sha256-X+mTF7Fc4FL/Nyt8ejvsWLwmNWIDXyYKCg00mdyEWhA=";
+
+    nativeBuildInputs = [
+      autoPatchelfHook
+    ];
+
+    dontAutoPatchelf = true;
+
+    preBuild = ''
+      			autoPatchelf node_modules/sass-embedded-linux-x64/dart-sass/src/dart
+      		'';
+
+    postBuild = ''
+      			cp -r ./ui/dist/ $out
+      		'';
+  };
+
+  postPatch = ''
+    		substituteInPlace crates/tauri-app/tauri.conf.json \
+    			--replace-warn '"frontendDist": "../../ui/dist"' '"frontendDist": "${frontend}"'
+    		substituteInPlace crates/tauri-app/tauri.conf.json \
+    			--replace-warn '"npm run build"' '""'
+    	'';
```

Comment:
mixed tabs and spaces

Reply:
nixfmt won't touch whitespace in strings. That's why it kept the mixed tabs and spaces in this string block

Reply:
TIL